docs-src/docker.md

```diff
@@ -0,0 +1,89 @@
+# Docker for Kakunin tests
+
+This section will walk you through building your own docker image for your tests.
+
+## Dockerfile:
+
+This file is responsible for building the whole environment for our e2e tests.
+It will allow you to run tests on local and CI environments, 
+by configuring and copying the whole project inside the container.
+
+To clarify what's happening Dockerfile example contains comments describing the whole process.
+
+### Example:
```

Comment:
Maybe add info that this is an example of `Dockerfile`. Maybe it's obvious but it would make it much easier to navigate and analyse the doc.